### PR TITLE
E2e

### PR DESCRIPTION
**Pull request by KarolDenst**

### CODE REVIEW

**Review by LeszekKrz — APPROVED** [review]

File: frontend/src/app/core/functions/get-role.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function getRole(): string {
```
Comment (Marvvvin5) [inline frontend/src/app/core/functions/get-role.ts:1]: where is it set? where is it used? How can you tell the role of user based on random token?

File: frontend/src/app/core/services/user.service.ts [hunk 2]
```diff
@@ -61,7 +63,7 @@ export class UserService {
     return this.httpClient.put<void>(`${this.registrationPageWebAPIUrl}/user`, updateUserDTO, httpOptions);
   }
 
-  loginUser(userForLogin: UserForLoginDTO): Observable<string> {
-    return this.httpClient.post<string>(`${this.registrationPageWebAPIUrl}/login`, userForLogin);
+  loginUser(userForLogin: UserForLoginDTO): Observable<AuthenticationResponseDTO> {
```
Comment (Marvvvin5) [inline frontend/src/app/core/services/user.service.ts:66]: if it is changed here does it work with be? When changing the be you can also change the way auth works. So Bearer gets added on fe not be

Comment (KarolDenst) [reply frontend/src/app/core/services/user.service.ts:66]: This works with BE. Before we used to get a string and then cast it to AuthenticationResponseDTO which didn't make much sense so i changed it

File: frontend/src/app/core/functions/get-role.ts (hunk 1 above)
Comment (KarolDenst) [reply frontend/src/app/core/functions/get-role.ts:1]: it is set after logging in. We send request to Get user date endpoint and save it

**Pull request merged**